README.md

```diff
@@ -178,4 +183,27 @@ Article.objects.all().select_json(author_name="meta__author__name")\
 
 
 
-#####For more information about raw jsonb operation, please see [PostgreSQL Documentation](http://www.postgresql.org/docs/9.4/static/functions-json.html)
+support geo search in jsonb
+===========================
+
+**require**: postgresql plugin: 
+
+1. cube
+2. earthdistance
+
+Demo
+
+```python
+Article.objects.filter(data__location__near=[39.9, 116.4,5000]) # longitude，latitude，search range
+```
+
+or 
+
+```
+Article.objects.filter(data__location__near='39.9,116.4,5000') # latitude，longitudesearch range
+```
+
+**Alert**: if you don't pass exact number of params, this filter will not be used
+
```

Comment:
Maybe also some advice about how to create index on this specific use, if it possible.